README.md

```diff
@@ -42,43 +42,43 @@ Various relevant PRISM models (mostly conversions of the SBML files in `./SBML`)
 
 This circuit is part of the genetic circuits designed by the software tool Cello [1]. The circuit has three input arguments IPTG, aTc, and Ara and one output argument YFP.
 
-![Figure1](https://github.com/fluentverification/CaseStudies_StochasticModelChecking/blob/refactor/Figures/0x8E_Original.png "Figure 1")
+![Figure1](/Figures/0x8E_Original.png "Figure 1")
 
 In the laboratory, circuit 0x8E showed an unexpected, glitching behavior. Further analysis resulted in two more designs of the circuit as well as the analysis of its glitching behavior [2].
 
-![Figure2](https://github.com/fluentverification/CaseStudies_StochasticModelChecking/blob/refactor/Figures/0x8E_LogicHazardFree.png "Figure 2")
+![Figure2](/Figures/0x8E_LogicHazardFree.png "Figure 2")
```

Comment:
Change to relative path `![Figure2](./Figures/0x8E_LogicHazardFree.png "Figure 2")`